testing/unittests/jlibtestsCustomPTree.xml

```diff
@@ -0,0 +1 @@
+<root/>
```

Comment:
I am not sure this should be checked in - normally unit tests would create any files that they then read.